packages/cli/src/metrics/license-metrics.service.ts

```diff
@@ -30,6 +31,7 @@ export class LicenseMetricsService {
 			{ name: 'totalUsers', value: totalUsers },
 			{ name: 'totalCredentials', value: totalCredentials },
 			{ name: 'productionExecutions', value: productionExecutions },
+			{ name: 'billableProductionExecutions', value: billableProductionExecutions },
```

Comment:
It's been called `productionRootExecutions` in the licenses. Do you think we should change it?

```suggestion
			{ name: 'productionRootExecutions', value: billableProductionExecutions },
```